test/testPlotWidget.cc

```
@@ -86,5 +87,447 @@ namespace minsky
        // width change will be proportionate - just check the new height is about right
       EXPECT_NEAR(legendHeight+20,newLegendHeight,2);
     }
+
+
+
+    TEST_F(PlotWidgetTest, numLines)
```

Clang (14.0.6)

[error] all parameters should be named in a function

(readability-named-parameter,-warnings-as-errors)